salt/orchestrate/aws/map_templates/ike_edx.yml

```diff
@@ -1,4 +1,4 @@
-edx:
+ike_edx:
```

Comment:
This needs to stay the same. The identifier here is used to determine which cloud profile will be deployed.